package.json

```diff
@@ -56,7 +56,7 @@
     "scripts": {
         "generate-regex": "node tools/generate-identifier-regex.js",
 
-        "test": "node test/run.js && npm run lint && npm run coverage",
+        "test": "if [ -d test ]; then node test/run.js && npm run lint && npm run coverage; fi",
```

Comment:
Perhaps have a top-level `unit-tests.js` or something similar? This one will first check if `test/run.js` exists and run it. Otherwise, do nothing.